backend/src/impl/db_models/system_metadata_model.py

```diff
@@ -101,34 +140,43 @@ def from_dict(cls, dikt) -> SystemModel:
                 document["dataset"] = None
             dikt.pop("dataset_metadata_id")
 
+        metric_stats = document["metric_stats"]
```

Comment:
I missed this bug ealier. I think this line is causing new systems submissions to fail. We probably need to fix this before depolying.

Reply:
Ah I should've tested the submission. Thanks!